samples/test/util.py

```diff
@@ -535,3 +539,14 @@ def _parse_parameters(execution: metadata_store_pb2.Execution) -> dict:
         if name.startswith('output:'):
             parameters['outputs'][name[len('output:'):]] = raw_value
     return parameters
+
+
+def disable_cache():
+    def _disable_cache(task):
```

Comment:
If you don't need to parameterize anything, you don't need the outer function. 
See this for example: https://github.com/kubeflow/pipelines/blob/master/samples/core/pipeline_transformers/pipeline_transformers.py

Reply:
done